### PR TITLE
Make concise work

### PR DESCRIPTION
**Pull request by Ajnasz**
--concise parameter were not passed to tester and there was a check if it's true, which should check check to false I guess.

### CODE REVIEW

**Comment by paazmaya** [comment]
Not exactly sure how `concise` is supposed to affect the tests, but since by default it is `false` the given changes looks legitimate.

**Comment by mickaelandrieu** [comment]
@Ajnasz would you mind to rebase master branch please ? 
Sounds good, but unmergeable.

**Comment by Ajnasz** [comment]
I rebased and force pushed it, but one build failed. I'm not sure, the issue is related.

**Comment by mickaelandrieu** [comment]
I relaunch the build, ping /c @paazmaya can you review this one too ?

I'll merge with your :+1:

File: modules/tester.js [hunk 1]
```diff
@@ -1063,7 +1063,7 @@ Tester.prototype.begin = function begin() {
         config = getConfig([].slice.call(arguments)),
         next = function() {
             config.test(this, this.casper);
-            if (this.options.concise)
+            if (!this.options.concise)
```
Comment (paazmaya) [inline modules/tester.js]: Perhaps curly brackets could be used here, as it seems to be the case with the rest of the file

**Comment by paazmaya** [comment]
Looks good, so all I can say is :+1:

File: modules/tester.js (hunk 1 above)
Comment (mickaelandrieu) [inline modules/tester.js]: For consistency, I agree.

Then, let's go for merge (I will probably update docs)

Comment (paazmaya) [inline modules/tester.js]: I think the merge could be done now, and the curly brackets consistency added later.

Comment (Ajnasz) [inline modules/tester.js]: Well, I can add braces, but this file has some other if statements without braces.
What is your current styleguide? Braces for every if statement?

Comment (paazmaya) [inline modules/tester.js]: Braces for everything would seem to be the future approach.

Comment (Ajnasz) [inline modules/tester.js]: I added the braces, but now jshint breaks the build :)
Should I go through on those errors?

Comment (paazmaya) [inline modules/tester.js]: It is about the given file having too many statements.

I am currently working on #1133 which touches this, so you can leave it as it is.

**Comment by mickaelandrieu** [comment]
```
OK
jshint .
modules/tester.js: line 1031, col 40, This function has too many statements. (18)
```
this is related.

**Comment by Ajnasz** [comment]
I reverted most of the braces I added, because the jshint version looks buggy for me, see the commit message in 27ac95e346edb

**Comment by mickaelandrieu** [comment]
Ok. Squash your commits and I'll merge it: we shouldn't waste "too much" time to please js(h/l)int...

**Comment by Ajnasz** [comment]
Squashed it.

**Comment by mickaelandrieu** [comment]
Merged, thank you @Ajnasz !

**Pull request merged**